### ISSUE TITLE
build-push action generates unusable container

### ISSUE THREAD

**Issue by dimalvovs** [DeshpandeLab/SpaceMarkers#31]
To some reason the container that is built using the buld-push action generates unusable container. 

```
docker run ghcr.io/deshpandelab/spacemarkers:sha256-2860bdf5c3b7afaac12237414fcc25b6942695faf27610535a2364d2e5e81fee     

Unable to find image 'ghcr.io/deshpandelab/spacemarkers:sha256-2860bdf5c3b7afaac12237414fcc25b6942695faf27610535a2364d2e5e81fee' locally
sha256-2860bdf5c3b7afaac12237414fcc25b6942695faf27610535a2364d2e5e81fee: Pulling from deshpandelab/spacemarkers
docker: unsupported media type application/vnd.oci.empty.v1+json.
```

Addressing the container by it's branch works though:
```
docker run ghcr.io/deshpandelab/spacemarkers:main                                                                   

Unable to find image 'ghcr.io/deshpandelab/spacemarkers:main' locally
main: Pulling from deshpandelab/spacemarkers
9b857f539cb1: Already exists 
723d93f4c3a7: Already exists 
6213efcac37a: Already exists 
40cc9f7250e6: Already exists 
...
```

**Comment by dimalvovs** [comment]
addressed in 533416ac0099f2a1ed0dac0feecee6a8973448f2

**Comment by dimalvovs** [comment]
also need to use `@sha` instead
